.vib/postgresql/goss/goss.yaml

```diff
@@ -1,29 +1,48 @@
 command:
-  postgresql-connect:
-    exec: PGPASSWORD="$POSTGRES_PASSWORD" psql -U "$POSTGRES_USER" -d "$POSTGRES_DB"
+  {{- $dbUser := .Vars.auth.username }}
+  {{- $dbPassword := .Vars.auth.password }}
+  {{- $dbName := .Vars.auth.database }}
+  {{- $lb_port := .Vars.primary.service.ports.postgresql }}
+  {{- $port := .Vars.containerPorts.postgresql }}
+  {{- $replica_nodes := .Vars.readReplicas.replicaCount }}
+  {{- $table := printf "test_%s" (randAlpha 5) }}
+  postgresql-check-replication:
+    exec: export PGPASSWORD='{{ $dbPassword }}' && psql -U '{{ $dbUser }}' -d '{{ $dbName }}' -p {{ $lb_port }} -h postgresql-{{ .Vars.primary.name }} -c "CREATE TABLE {{ $table }} (id int, first_name VARCHAR(50));" && sleep 1 {{ range $e, $i := until $replica_nodes }} && psql -U '{{ $dbUser }}' -d '{{ $dbName }}' -p {{ $port }} -h postgresql-{{ $.Vars.readReplicas.name }}-{{ $i }}.postgresql-{{ $.Vars.readReplicas.name }}-hl -c "SELECT * FROM {{ $table }};"{{ end }}
```

Comment:
Could we use a variable for `postgresql-{{ $.Vars.readReplicas.name }}`? It seems to be repeated several times in the first two tests.